Add bare-bones readme (#93)
gradient-ai/Graphcore-HuggingFace: README.md

# Training HuggingFace 🤗 Models on an IPU using Paperspace Gradient

Whether you are looking to generate images with Stable Diffusion, derive insights from text, or need to recognize audio samples, the examples in here have you covered. And by using Paperspace to access IPUs you can be up and running in minutes!

> See the [README notebook](README_first.ipynb) for more detail on the tasks which are covered.

Reviewer: This link distracts from the run on gradient button below, why have both?

Author: ```suggestion
```

Author: Good point

> ## Getting Started with our interactive notebooks
> 
> This repository contains notebooks which are meant to be run on Paperspace Gradient's IPU Machines:
> 

Reviewer: ```suggestion
Run the notebooks in this repository on our configured Paperspace Gradient's IPU Machines:
```
This feels more actionable to me because this readme is small, and the point is to get the user straight to the runtime

> [![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/3BFkBCt)

Author: ```suggestion
```

> [![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/3BFkBCt)

Author: No

# Training HuggingFace 🤗 Models on an IPU using Paperspace Gradient

Whether you are looking to generate images with Stable Diffusion, derive insights from text, or need to recognize audio samples, the examples in here have you covered. And by using Paperspace to access IPUs you can be up and running in minutes!

> Run the notebooks in this repository on our configured Paperspace Gradient's IPU Machines:

Author: ```suggestion
Run the notebooks in this repository on our ready-to-use IPU Machines in Paperspace Gradient:
```
